src/main/java/net/dv8tion/jda/core/requests/restaction/MessageAction.java

```diff
@@ -49,6 +49,8 @@
  * <br>This can be used to remove existing embeds from a message:
  * <br>{@code message.editMessage("This message had an embed").override(true).queue()}
  *
+ * <p><u>When this RestAction has been executed all provided files will be closed.</u></p>
```

Comment:
I would rephrase this to something like: "If you provide Files to this builder but decide not to actually send this message, use clearFiles() to clean up their resources. See description of sendFile(File)"